src/test-utils/index.tsx

```diff
@@ -57,6 +60,22 @@ const DEVICES: DeviceCollection = {
             isLandscape: false,
         },
     },
+    [MOBILE_DEVICE_IOS_INSET]: {
```

Comment:
I thought I'd find it at handy having a default device to choose from to simulate the inset at handy.
This is an initial proposal and it surely depends on your point of view/what do you want to accomplish with this etc. Since I surely don't know all the scenarios we could fall into, share your best generic "inset" device case we should consider for testing purposes here, and I'll update it!

For instance: what about the upper inset? I wouldn't simulate it just because the pages within the app consider the top safe area already, in some way. But much more scenarios could need to encompass that.